data/models

```diff
@@ -13,3 +13,4 @@ VivoBookS15X530UA|1335|55 57 59 61 63 65 67 69
 X580VD|1335 1431|45 50 55 60 65 70 75 80
 TUFGamingFX505GMFX505GM|1335 1431|45 50 55 60 65 70 75 80
 VivoBook15ASUSLaptopX542UF|1335|40 45 50 55 59 63 65 67
+VivoBook15ASUSLaptopX507UF|1335|55 60 62 65 68 72 76 80
```

Comment:
In the #73 you mentioned `55 60 65 70 75 80 85 90` as `get-temps`, you have different temps here. Please clarify, which ones you want to use as default for your laptop model.